bazel/README.md

```diff
@@ -181,13 +188,18 @@ for how to update or override dependencies.
     The TMPDIR path and MSYS2 `mktemp` command are used frequently by the `rules_foreign_cc`
     component of Bazel as well as Envoy's test scripts, causing problems if not set to a path
     accessible to both Windows and msys commands. [Note the `ci/windows_ci_steps.sh` script
-    which builds envoy and run tests in CI) creates this symlink automatically.]
+    which builds envoy and run tests in CI creates this symlink automatically.]
 
     In the MSYS2 shell, install additional packages via pacman:
     ```
     pacman -S diffutils patch unzip zip
     ```
 
+    Check that the PATH variable in the MSYS2 shell is not truncated because of spaces:
+    ```
+    $PATH
```

Comment:
I actually have no idea what this does or what the intent is above, could you explain? Maybe a copy paste quirk?

Reply:
(Since you clearly state to install msys2 to a path without spaces, this 'check' should probably be dropped entirely.)

Reply:
Sure.